src/ripple/app/ledger/LedgerConsensus.h

```diff
@@ -54,6 +54,37 @@ class LedgerConsensus
 
     virtual bool peerPosition (LedgerProposal::ref) = 0;
 
+    /** Apply open ledger transactions to a view.
+
+        Preconditions:
+
+            Caller must have ownership of the master mutex.
```

Comment:
The master mutex - as in the global one?

I hate to introduce new dependencies on it - I thought we were breaking it up... isn't there a smaller mutex that would do the job?